api/Element.json

```diff
@@ -1171,66 +2770,66 @@
             }
           },
           "status": {
-            "experimental": false,
+            "experimental": true,
```

Comment:
I would not have marked this as experimental. There seems to be a reasonably mature spec with unprefixed support in 3 browser engines. OTOH the related pointer lock APIs (e.g. [Document.exitPointerLock](https://developer.mozilla.org/en-US/docs/Web/API/Document/exitPointerLock)) [seem to be marked experimental](https://github.com/mdn/browser-compat-data/blob/master/api/Document.json#L3382), so at least it's consistent with those.